lib/bundle/brew_dumper.rb

```diff
@@ -75,6 +82,7 @@ def formula_inspector(f)
       {
         :name => f["name"],
         :full_name => f["full_name"],
+        :aliases => f["aliases"],
```

Comment:
Beside the alias, we need to store `oldname` as well, which however may complicate implementation.